packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -1110,13 +1110,21 @@ class ListView extends BoxScrollView {
   /// The `itemBuilder` callback will be called only with indices greater than
   /// or equal to zero and less than `itemCount`.
   ///
-  /// The `itemBuilder` should always return a non-null widget, and actually
-  /// create the widget instances when called. Avoid using a builder that
-  /// returns a previously-constructed widget; if the list view's children are
-  /// created in advance, or all at once when the [ListView] itself is created,
-  /// it is more efficient to use the [ListView] constructor. Even more
-  /// efficient, however, is to create the instances on demand using this
-  /// constructor's `itemBuilder` callback.
+  /// {@template flutter.widgets.ListView.builder.itemBuilder}
+  /// It is legal for `itemBuilder` to return `null`. If it does, the scroll view
+  /// will stop calling `itemBuilder`, even if it has yet to reach `itemCount`.
+  /// By returning `null`, the max scroll extent will not be accurate
+  /// unless the user reaches the end of the scroll view. This can cause the
+  /// scrollbar to grow as the user scrolls.
```

Comment:
Breadcrumbs would be helpful here:

```suggestion
  /// By returning `null`, the [ScrollPosition.maxScrollExtent] will not be accurate
  /// unless the user has reached the end of the [ScrollView]. This can also cause the
  /// [Scrollbar] to grow as the user scrolls.
```